python/lsst/skymap/healpixSkyMap.py

```diff
@@ -33,6 +34,9 @@
 from .tractInfo import TractInfo
 
 
+# TODO: Remove with DM-44799
+@deprecated("angToCoord has been deprecated and will be removed after v29.",
```

Comment:
Deprecation timer is shorter than you remember: this and all other occurrences should be `v28` (it's supposed to match the `version=` value).